src/containers/Projects/CreateProjectPopup/index.js

```diff
@@ -286,9 +287,7 @@ let CreateProjectPopup = (props) => {
 
               <div
                 className="gallery"
-                onClick={() => {
-                  openFile.current.click();
-                }}
+                onClick={openFile.current.click}
```

Comment:
I think I reverted it on my commit
It was throwing error on page refresh when I change in this way

Reply:
kindly check on yr local and revert this change if it is breaking